audio_clip_encoder.py

```diff
@@ -30,36 +29,35 @@ def __init__(self,
         self.aclp = AudioCLIP(pretrained=model_path)
```

Comment:
here u should set `self.aclp.eval()`

Reply:
should I override method AudioCLIP:eval so that it calls self.audio.eval() ?

Reply:
no, `nn.Module` eval should know that this is done. But anyway, u should change it from the outside. You should not change the code that is not ours because in other versions it may change

Reply:
or just use self.aclp.audio.eval() ?